build/Targets/Packages.props

```diff
@@ -58,7 +58,7 @@
     <MicrosoftNetCompilersVersion>2.6.0</MicrosoftNetCompilersVersion>
     <MicrosoftNetRoslynDiagnosticsVersion>2.6.0-beta2</MicrosoftNetRoslynDiagnosticsVersion>
     <MicrosoftNetCoreILAsmVersion>2.0.0</MicrosoftNetCoreILAsmVersion>
-    <MicrosoftNETCoreCompilersVersion>2.6.0-beta3-62308-01</MicrosoftNETCoreCompilersVersion>
+    <MicrosoftNETCoreCompilersVersion>2.8.0-beta2-62712-07</MicrosoftNETCoreCompilersVersion>
```

Comment:
This is necessary because the new CLI is now pushing properties like "EmbedAllSources". This is not recognized by the 2.6 version of the compiler and hence results in a build error. Need to have a base version of the compilers that supports all the properties being push here.

Reply:
@jaredpar Are we setting langversion somewhere so we don't depend on things newer than what our VS supports?

Reply:
Aside: it feels to me like the nupkg should be overriding the core C# targets so that this version mismatch can't happen.

Reply:
@jasonmalinowski yes it is controlled via LangVersion in Settings.props

Reply:
@nguerrera I'm also a bit confused about this.

Reply:
Why does `MicrosoftNETCoreCompilersVersion` not carry/use corresponding core C# targets that are used in place of what shipped with the CLI?

Reply:
@nguerrera it does. It's unclear why that's not working though. I've been going through this code as part of my other PR though. I think we may have some ordering issues.

Reply:
Oh, I misread "I'm also a bit confused by this" as "I'm confused by your comment and not I'm also confused that this doesn't already work. ;)